cli.go

```diff
@@ -1170,6 +1449,27 @@ func GithubListReposByLanguage(owner string, lang string) ([]*github.Repository,
 
 	return repos, nil
 }
+func ListAllReposByLanguage(lang string) ([]*github.Repository, error) {
+	lang = strings.TrimSpace(lang)
+
+	opts := &ghc.ListAllReposByLanguageOpts{
+		Language:     lang,
+		ExcludeForks: true,
+		// Limit:        limit, // Maybe add a `--limit=n` flag?
+	}
+	repos, err := ghClient.ListAllReposByLanguage(opts)
+	if err != nil {
+		return nil, err
+	}
+
+	return repos, nil
+}
+func GithubListReposByMetaSearch(query string) ([]*github.Repository, error) {
+	return ghClient.SearchRepos(query)
+}
+func GithubListReposByCodeSearch(query string) ([]*github.Repository, error) {
```

Comment:
Should this function iterate over the result of `ghClient.SearchRepos(query)` (since the result come as `CodeResult`) and return a list of the `Repository` type? Perhaps is easier to make `gh-client` return that type already.

Reply:
Well, as you suggested `gh-client` to return `CodeResult`, `GithubListReposByCodeSearch` now iterates over that result and gets `Repository` :)